scripts/docker/tgi/tgi-run.yaml

```diff
@@ -0,0 +1,46 @@
+version: '2'
+built_at: '2024-10-08T17:40:45.325529'
+image_name: local
+docker_image: null
+conda_env: local
+apis:
+- shields
+- agents
+- models
+- memory
+- memory_banks
+- inference
+- safety
+providers:
+  inference:
+  - provider_id: remote::tgi
```

Comment:
Can provider_id be a regular string instead of duplicating provider_type?

Reply:
Yeah, updated to be arbitrary provider_id.